src/client/components/TablePaginated/Filters/Country/Country.scss

```diff
@@ -3,3 +3,42 @@
 div.filter-country {
   width: 235px;
 }
+
+.tooltip-container {
+  .react-tooltip.filter-country__tooltip {
+    max-width: unset;
+
+    .regions-container {
+      column-gap: $spacing-xs;
+      display: grid;
+      font-size: $font-xs;
+      max-height: 500px;
+      max-width: 1200px;
+      overflow-y: scroll;
+      pointer-events: auto;
+      row-gap: $spacing-xxxs;
+
+      .region-title {
+        background-color: lighten($ui-accent, 55%);
+        font-weight: bold;
+        padding-bottom: $spacing-xxxs;
+        position: sticky;
+        top: 0;
+      }
+
+      .countries-container {
+        display: grid;
+        gap: $spacing-xxxs;
+        height: min-content;
+
+        .country {
+          height: $spacing-s;
+          max-width: 170px;
```

Comment:
This makes the width more consistent when certain options are selected. 

For example, instead of this:

https://github.com/user-attachments/assets/fe4b184a-ce30-4a59-a10a-c9d8b685348e

It looks like this:


https://github.com/user-attachments/assets/5baf9f37-f23c-4eae-8962-1a4795b57cf7